FirstClass.java

```diff
@@ -19,10 +29,13 @@ public static void main(String[] args) {
         }
         System.out.println("The quantity of 10-digit numbers, which have different numerals = 9*9! = " + 9 * factorial9);
 
-        //calculation of the quantity of 10-digit numbers, which have at least 2 identical numerals
+        /* calculation of the quantity of 10-digit numbers, which have at least 2 identical numerals
+        to solve the task - from the quantity of 10-digit numbers we should deduct
+        the quantity of 10-digit numbers, which have different numerals
+         */
         long Quantity10DigitNumber2ident = Quantity10DigitNumbers - 9 * factorial9;

```

Comment:
Same naming problems as above. Names should be easily readable and as much self-explanatory as possible.